contrib/thinkfan/files/thinkfan

```diff
@@ -0,0 +1,9 @@
+# thinkfan service
+
+type            = bgprocess
```

Comment:
can we make this a process service instead?

Reply:
kinda, it wouldn't write to syslog in that case but I guess that's fine

Reply:
can always just log to a file